tests/functional/history/test_list.py

```diff
@@ -61,28 +61,28 @@ def test_show_one_call_present(self):
         self.history_recorder.record('CLI_RC', rc, 'CLI')
         self.run_cmd('history list', expected_rc=0)
         self.assertIn(b'ec2 describe-regions', self.binary_stdout.getvalue())
-        def test_multiple_calls_present(self):
-            self.parsed_responses = [
-                {
-                    "Regions": [
-                        {
-                            "Endpoint": "ec2.ap-south-1.amazonaws.com",
-                            "RegionName": "ap-south-1"
-                        },
-                    ]
-                },
-                {
-                    "UserId": "foo",
-                    "Account": "bar",
-                    "Arn": "arn:aws:iam::1234567:user/baz"
-                }
-            ]
-            _, _, rc = self.run_cmd('ec2 describe-instances', expected_rc=0)
-            self.history_recorder.record('CLI_RC', rc, 'CLI')
-            _, _, rc = self.run_cmd('sts get-caller-identity', expected_rc=0)
-            self.history_recorder.record('CLI_RC', rc, 'CLI')
-            self.run_cmd('history list', expected_rc=0)
-            self.assertIn(b'ec2 describe-regions',
-                          self.binary_stdout.getvalue())
-            self.assertIn(b'sts get-caller-identity',
-                          self.binary_stdout.getvalue())
+    def test_multiple_calls_present(self):
+        self.parsed_responses = [
+            {
+                "Regions": [
+                    {
+                        "Endpoint": "ec2.ap-south-1.amazonaws.com",
+                        "RegionName": "ap-south-1"
+                    },
+                ]
+            },
+            {
+                "UserId": "foo",
+                "Account": "bar",
+                "Arn": "arn:aws:iam::1234567:user/baz"
+            }
+        ]
+        _, _, rc = self.run_cmd('ec2 describe-instances', expected_rc=0)
+        self.history_recorder.record('CLI_RC', rc, 'CLI')
+        _, _, rc = self.run_cmd('sts get-caller-identity', expected_rc=0)
+        self.history_recorder.record('CLI_RC', rc, 'CLI')
+        self.run_cmd('history list', expected_rc=0)
+        self.assertIn(b'ec2 describe-regions',
```

Comment:
Looks like this should assert that `ec2 describe-instances` was called. Can you fix this (squash your commits, please)? I can assist as well!

Reply:
Hi @kdaily , sorry for the late answer, I didn't see the notification of your review. I made the modification and squashed the commits. I hope I didn't miss nothing.